cmd/manager/main.go

```diff
@@ -13,15 +13,14 @@ import (
 	"github.com/operator-framework/operator-marketplace/pkg/migrator"
 
 	apiconfigv1 "github.com/openshift/api/config/v1"
-	olm "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	ofapi "github.com/operator-framework/api/pkg/operators/v1alpha1"
```

Comment:
nit: I think the alias of the package should be `v1alpha1` or `operatorsv1alpha1` per our import convention in OLM and k8s in general. See the OLM e2e tests for examples of this. For example, in the catalog source e2e tests catalogs are simply `v1alpha1.CatalogSource`

Reply:
@exdx I wanted to distinguish between marketplace v1 and v2 and olm's v1alpha1. I think `operatorsv1alpha1` does the job too so I'll switch this over to `operatorsv1alpha1`